### PR TITLE
Update LIP 46 for strings and substore

### PR DESCRIPTION
**Pull request by gkoumout**

### CODE REVIEW

File: proposals/lip-0046.md [hunk 1]
```diff
@@ -49,8 +47,8 @@ We define the following constants:
 | **Name**                       | **Type** | **Value**       | **Description**                                |
 |--------------------------------|----------|-----------------|------------------------------------------------|
 | **Constants**                  |          |                 |                                                |
-| `MODULE_ID_RANDOM `            | bytes    | TBD             | ID of the Random module                        |
-| `STORE_PREFIX_RANDOM`          | bytes    | `0x0000`        | Prefix of the random substore                  |
+| `MODULE_NAME_RANDOM `          | string   | "random"        | Module's name                                  |
+| `SUBSTORE_PREFIX_RANDOM`       | bytes    | `0x0000`        | Prefix of the random substore                  |
```
Comment (ricott1) [inline proposals/lip-0046.md:51]:
```suggestion
| `SUBSTORE_PREFIX_RANDOM`       | bytes    | `0x0000`        | Prefix of the random substore.                  |
```

File: proposals/lip-0046.md [hunk 2]
```diff
@@ -49,8 +47,8 @@ We define the following constants:
 | **Name**                       | **Type** | **Value**       | **Description**                                |
 |--------------------------------|----------|-----------------|------------------------------------------------|
 | **Constants**                  |          |                 |                                                |
-| `MODULE_ID_RANDOM `            | bytes    | TBD             | ID of the Random module                        |
-| `STORE_PREFIX_RANDOM`          | bytes    | `0x0000`        | Prefix of the random substore                  |
+| `MODULE_NAME_RANDOM `          | string   | "random"        | Module's name                                  |
```
Comment (ricott1) [inline proposals/lip-0046.md:50]:
```suggestion
| `MODULE_NAME_RANDOM `          | string   | "random"        | Module's name.                                  |
```

**Review by ricott1 — APPROVED** [review]

File: proposals/lip-0046.md [hunk 3]
```diff
@@ -38,9 +38,7 @@ It is worth noting that random seed computation from this commit and reveal proc
 
 ## Specification
 
-In this section, we specify the substores that are part of the Random module store, and the protocol logic called during the block processing and genesis block processing.
-
-The Random module has module ID `MODULE_ID_RANDOM` (see the table below). It is a module with no dependencies.
+In this section, we specify the substores that are part of the Random module store, and the protocol logic called during the block processing and genesis block processing. It is a module with no dependencies.
```
Comment (sergeyshemyakov) [inline proposals/lip-0046.md:41]: I would still say that the Random module has name `MODULE_NAME_RANDOM`.

**Review by sergeyshemyakov — APPROVED** [review]
All good, just a small remark from my side.

File: proposals/lip-0046.md (hunk 3 above)
Comment (gkoumout) [reply proposals/lip-0046.md:41]: Not sure is necessary, but ok, added

**Review by karmacoma — APPROVED** [review]

**Pull request merged**